packages/laravel/src/Commands/PrintConfigurationCommand.php

```diff
@@ -10,7 +10,7 @@
 
 class PrintConfigurationCommand extends Command
 {
-    protected $signature = 'hybridly:config {--pretty}';
+    protected $signature = 'hybridly:config {--pretty} {buildPath?}';
```

Comment:
`buildPath` here is inconsistent with `basePath` used in the other changes. I'd rather have `basePath` I think.

Reply:
Indeed thanks for the review